plugins/power/meson.build

```diff
@@ -41,6 +41,14 @@ gsd_power = executable(
   install_dir: gsd_libexecdir
 )
 
+led_breathe = executable(
+  'led-breathe',
+  files('led-breathe.c'),
+  install: true,
```

Comment:
Could also add `install_mode: …` (from [Meson 0.47](http://mesonbuild.com/Reference-manual.html#executable)) to add the setuid bit, rather than relying on the Debian packaging to do it.

Reply:
Please test that carefully if you do take that approach - I think the debian packaging might strip the setuid bit unless explicitly told about it.

Reply:
For the avoidance of doubt, I meant to have the setuid stuff in *both* the `meson.build` and the `debian/blahblah` packaging.

Reply:
Kept the packaging bits, added meson bits.

Reply:
Turns out, the meson bits were necessary, and not including them would have been a regression.